CONTRIBUTING.md

```diff
@@ -26,6 +27,132 @@ The easier it is for us to reproduce it, the faster it'll be fixed!
 
 Please don't include any private/sensitive information in your issue!
 
+## Hack On Libpod
+
+This section describe to you how to start to hack on libpod.
+
+### Prepare your environment
+
+Assume that you use Fedora 29, before starting to hack you need to install some requirements:
+```shell
+sudo dnf install -y \
+    podman \
+    atomic-registries \
+    btrfs-progs-devel \
+    conmon \
+    containernetworking-cni \
+    device-mapper-devel \
+    git \
+    glib2-devel \
+    glibc-devel \
+    glibc-static \
+    go \
+    golang-github-cpuguy83-go-md2man \
+    gpgme-devel \
+    iptables \
+    libassuan-devel \
+    libgpg-error-devel \
+    libseccomp-devel \
+    libselinux-devel \
+    make \
+    ostree-devel \
+    pkgconfig \
+    runc \
+    skopeo-containers
+```
+
+For [other distos you can find more available commands](install.md) on the install documentation to prepare your environment.
```

Comment:
distos or distros?

Reply:
I am late to the game, sorry. Why not just point to `install.md` for those packages? It would make the document more vendor/distribution neutral and avoid redundancy.

Reply:
I agree :)